web/app.js

```diff
@@ -72,6 +72,7 @@ import { PDFPresentationMode } from "./pdf_presentation_mode.js";
 import { PDFSidebarResizer } from "./pdf_sidebar_resizer.js";
 import { PDFThumbnailViewer } from "./pdf_thumbnail_viewer.js";
 import { PDFViewer } from "./pdf_viewer.js";
+import { Scripting } from "../src/pdf.scripting_api.js";
```

Comment:
Sorry, but there's absolutely no way that this works correctly once the viewer has been built/packaged.

I'd suggest looking at how the `pdfjs-lib` property is used here in the `/web` folder, for inspiration.